cmd/oklog/ingest.go

```diff
@@ -330,7 +331,7 @@ func runIngest(args []string) error {
 			)))
 			registerMetrics(mux)
 			registerProfile(mux)
-			return http.Serve(apiListener, mux)
+			return http.Serve(apiListener, cors.Default().Handler(mux))
```

Comment:
This has [reasonable defaults](https://github.com/rs/cors#parameters), but we could also specify some of the options like allowed origins via cli flags. `AllowedOrigins` seems like the most often change that would need to happen.